composer.json

```diff
@@ -26,6 +26,11 @@
     },
     "require": {
         "php": "^7.2",
+        "ext-SimpleXML": "*",
```

Comment:
Does this have to be written this way? Why not using `ext-simplexml`, like in all other cases?